ui/pages/confirmations/components/confirm/info/token-transfer/transaction-flow-section.tsx

```diff
@@ -0,0 +1,55 @@
+import { TransactionMeta } from '@metamask/transaction-controller';
+import React from 'react';
+import { ConfirmInfoSection } from '../../../../../../components/app/confirm/info/row/section';
+import {
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+} from '../../../../../../components/component-library';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  IconColor,
+  JustifyContent,
+} from '../../../../../../helpers/constants/design-system';
+import { useConfirmContext } from '../../../../context/confirm';
+import { useDecodedTransactionData } from '../hooks/useDecodedTransactionData';
+import { ConfirmLoader } from '../shared/confirm-loader/confirm-loader';
+import { NameOrAddressDisplay } from './name-or-address-display';
+
+export const TransactionFlowSection = () => {
+  const { currentConfirmation: transactionMeta } =
+    useConfirmContext<TransactionMeta>();
+
+  const { value, pending } = useDecodedTransactionData();
```

Comment:
The downside of this is every token transfer will now trigger 4byte and Sourcify requests which previously was only done when the advanced toggle was pressed.

Reply:
That's true... This is the only way I can think of to extract the recipient address. 🤔